scripts/_events.js

```diff
@@ -1,30 +1,38 @@
-const hre = require('hardhat');
+const hre = require("hardhat");
+const path = require("path");
+
 const explorerOpUrl = "https://optimism-sepolia.blockscout.com/";
 const explorerBaseUrl = "https://base-sepolia.blockscout.com/";
+const configRelativePath = process.env.CONFIG_PATH || "config.json";
+const configPath = path.join(__dirname, "..", configRelativePath);
+const config = require(configPath);
+
+function filterChannelEvents(portAddress) {
```

Comment:
@kenobijon your original suggestion for channel events would have likely worked, but there's a chance someone manually works with the config in which case the sendPacket and createChannel fields do not necessarily equal one another

Reply:
that makes sense, using createChannel field explicitly seems best